abci/types/types.pb.go

```diff
@@ -1970,7 +1970,7 @@ type ResponseCheckTx struct {
 	Codespace string  `protobuf:"bytes,8,opt,name=codespace,proto3" json:"codespace,omitempty"`
 	Sender    string  `protobuf:"bytes,9,opt,name=sender,proto3" json:"sender,omitempty"`
 	Priority  int64   `protobuf:"varint,10,opt,name=priority,proto3" json:"priority,omitempty"`
-	// mempool_error is set by Tendermint.
+	// mempool_error is set by CometBFT.
```

Comment:
This was autogenerated right, the PR is not manually changing this?

Reply:
Yep, generated using make something.